common-workflow-language/cwl-v1.3
# refresh from 1.2.1-proposed

- Backport formatting fixes (#210)
- fix typo in ResourceRequirement.coresMin description (#214)
- fix workflow scatter typos (#215)
- Two tests were accidently still specifying v1.0 (#217)
- Add missing `label` field
- backport labels for some conditional tests
- Increase the timeout in the time limit test
- Create dependabot.yml
- Added conformance tests for File and Directory glob
- add hyperlinks to intent definition
- Bump rojopolis/spellcheck-github-actions from 0.23.0 to 0.29.0
- Bump actions/setup-python from 2 to 4
- Bump actions/checkout from 2 to 3
- ci: use cwltest pytest plugin for better speed
- Bump rojopolis/spellcheck-github-actions from 0.29.0 to 0.30.0
- Fixed MacOS X compatibility for wc-tool
- Document that ExpressionTool outputs are always valid
- add render script
- install cwltool when rendering
- also lockfile
- render: skip docker
- render: re-use git checkout
- official IANA media types for CWL documents
- spelling: check YAML docs
- Fixes #193
- Substitute URI with IRI in docs
- Add tests for decimal representation of very big and very small numbers
- ci: add codespell check
- sync metaschema
- typos
- Fix typo {} vs ()
- Bump rojopolis/spellcheck-github-actions from 0.30.0 to 0.32.0
- Bump codespell-project/actions-codespell from 1.0 to 2.0
- Update requirements.txt
- Update requirements.txt
- All loadContents tests are required
- default_with_falsey_value (#48)
- Bump rojopolis/spellcheck-github-actions from 0.32.0 to 0.33.0 (#257)
